src/prefix.rs

```diff
@@ -49,6 +51,20 @@ where
     de_fn_v: DeserializeVFn<T>,
 }
 
+impl<K, T> Debug for Prefix<K, T> 
+where
+    K: KeyDeserialize,
+    T: Serialize + DeserializeOwned,
+{
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("Prefix")
+         .field("storage_prefix", &self.storage_prefix)
+         .field("data", &self.data)
+         .field("pk_name", &self.pk_name)
+         .finish()
```

Comment:
```suggestion
         .finish_non_exhaustive() // Show that some other field(s) exist.
```

see https://doc.rust-lang.org/std/fmt/struct.DebugStruct.html#method.finish_non_exhaustive